specification/peering/resource-manager/Microsoft.Peering/preview/2019-03-01-preview/peering.json

```diff
@@ -0,0 +1,1449 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-03-01-preview",
+    "title": "PeeringManagementClient",
+    "description": "APIs to manage Peering resources through the Azure Resource Manager."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Peering/legacyPeerings": {
+      "get": {
+        "tags": [
+          "LegacyPeerings"
+        ],
+        "description": "Lists all of the legacy peerings under the given subscription matching the specified kind and location.",
+        "operationId": "LegacyPeerings_List",
+        "parameters": [
+          {
+            "name": "peeringLocation",
+            "in": "query",
+            "description": "The location of the peering.",
```

Comment:
is there any documentation that could be added on what values location should take?

Reply:
yes, we do. We expect that users will Get the avaliable peering locations first then use one the listed. Its somewhere around 200 peering locations and adding more each year.

Reply:
ok, you could evaluate updating your description for customers to better make use of it.